packages/components/forms/src/text-input/TextInput.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useCallback, useState, ChangeEvent } from 'react';
+import { BaseInput } from '@contentful/f36-inputs';
+import { Box, Flex } from '@contentful/f36-core';
+import { Label } from '@contentful/f36-forms';
+import { ValidationMessage } from '@contentful/f36-validation-message';
+import { HelpText } from '@contentful/f36-helptext';
+import { CopyButton } from '@contentful/f36-copybutton';
+import getStyles from './TextInput.styles';
+import { TextInputProps } from './types';
+
+export const _TextInput = (
+  {
+    className,
```

Comment:
Oh I forgot to talk about this

I don't see this `className` being used anywhere
I was actually going to ask "What should we do with className in these input/field components?"